google/cloud/spanner/samples/samples.cc

```diff
@@ -3802,6 +3802,16 @@ void QueryInformationSchemaDatabaseOptions(
 }
 // [END spanner_query_information_schema_database_options]
 
+// [START spanner_set_batching_delay]
+void CommitWithBatchingDelay(google::cloud::spanner::Client client,
```

Comment:
This example does not show me anything that I did not know from reading the prototype.  Do we really need it?

Can we change it to do something less obvious?

Reply:
In particular, `batching_delay` is unused.

Reply:
Updated.